package.json

```diff
@@ -55,14 +55,12 @@
     "ember-computed-decorators": "^0.2.2",
     "ember-cli-htmlbars": "^1.0.1",
     "ember-cli-sass": "^5.2.0",
-    "ember-frost-core": "^0.0.13",
     "ember-frost-theme": "^1.4.4",
     "ember-frost-css-core": "^0.2.1",
-    "ember-frost-icons": "^1.0.0",
-    "ember-block-slots": "^0.4.3",
+    "ember-frost-info-bar": "^1.0.0",
```

Comment:
is this part of the addon itself? or just dummy ap? It should be in blueprint if its part of the addon.